internal/kojiapi/server.go

```diff
@@ -106,7 +106,7 @@ func (server *Server) PostCompose(w http.ResponseWriter, r *http.Request) {
 		repositories := make([]rpmmd.RepoConfig, len(ir.Repositories))
 		for j, repo := range ir.Repositories {
 			repositories[j].BaseURL = repo.Baseurl
-			repositories[j].GPGKey = repo.Gpgkey
+			repositories[j].GPGKey = *repo.Gpgkey
```

Comment:
Hm this might be nil now, no? Not sure what happens in go when you deref nil.

Reply:
You are right, thanks! Fixed.